README.md

```diff
@@ -101,24 +101,24 @@ BibTeX Citation
 ---------------------
 
 ```
-@software{wei_kang_2023_7693957,
+@software{wei_kang_2024_10520458,
   author       = {Wei Kang and
                   Sergio Rey and
-                  Philip Stephens and
                   James Gaboardi and
+                  Philip Stephens and
                   Nicholas Malizia and
                   Stefanie Lumnitz and
                   Levi John Wolf and
                   Charles Schmidt and
                   Jay Laura and
                   Eli Knaap},
-  title        = {pysal/giddy: v2.3.4},
-  month        = mar,
-  year         = 2023,
+  title        = {pysal/giddy: v2.3.5},
+  month        = jan,
+  year         = 2024,
   publisher    = {Zenodo},
-  version      = {v2.3.4},
-  doi          = {10.5281/zenodo.7693957},
-  url          = {https://doi.org/10.5281/zenodo.7693957}
+  version      = {v2.3.5},
+  doi          = {10.5281/zenodo.10520458},
+  url          = {https://doi.org/10.5281/zenodo.10520458}
```

Comment:
Shall we use the "all version DOI" zenodo provides instead? [10.5281/zenodo.1322825](https://zenodo.org/doi/10.5281/zenodo.1322825)? That way this does not need to be updated every release.

Reply:
This is a good point. And if folks do cite the package (it does happen sometimes), having version-specific citations muddies the waters.

@weikang9009 I am concur with @martinfleis.

Reply:
Good idea! I have pushed a commit for this.